### DISCUSSION TITLE
AI牙刷

### DISCUSSION THREAD

**Discussion by Raywei0214** [1101-GS3073-C/Python-Community-GS3073-C#597] [General]
有再刷牙的各位應該都知道飛利浦這個牌子吧，雖然我是沒用過他們家的牙刷啦，但現在連牙刷都有AI的應用我覺得非常的厲害，還有各式各樣的功能，不看還不知道刷牙要去注意那麼細節，沒想到AI已經深入我們的日常了，不包括我的，因為我用普通牙刷，刷一刷會炸毛的那種。
https://www.momoshop.com.tw/category/DgrpCategory.jsp?d_code=2152100039&p_orderType=6&showType=chessboardType

**Comment by awkward-willy** [comment]
透過感測器收集數據，再經由 AI 做分析並給出建議感覺是未來商品的一大趨勢之一，如此的設計能讓我們更清楚掌握使用狀況，不需要每次都要請教專家的建議。只是在價格的方面可能還要努力調整至消費者能接受的價位。

**Comment by magicmicheal** [comment]
最近剛好遇到牙齒上的問題，想維護好口腔，是真的蠻費時費力的，但這項產品靠著與AI結合，按照每個人不同的口腔，達到了完完全全的客製化，應該能省下不少的心力

**Comment by awehao** [comment]
好扯歐，什麼鬼，感覺我會很需要用到這個，我刷牙都刷不好，哭阿

**Reply by titan92601** [reply]
這位同學，您的這則回覆看似是紅色的，但是不論是好的回覆或是不那麼好的回覆，都是能夠促進我們這個社群，真的要感謝老師給了我們這個機會，才能在這討論，也才譨在這找到知己，對吧浩浩。

**Comment by bww0401** [comment]
有時候刷牙常常會忽略後排牙齒，使用這個產品感覺可以改善我的刷牙習慣，但價位上似乎可以再做調整，畢竟這麼好的產品如果因為價格使得無法普及，那就太可惜了!

**Comment by manipulateed** [comment]
想不到AI都能結合牙刷了，但光電動牙刷或一般牙刷都能被我刷壞，希望在這個產品的使用上能增加壽命或是告知延長壽命的使用方式，我也很期待看到這個功能的普及，畢竟裝一口新牙齒可是很貴的。

**Comment by titan92601** [comment]
刷牙其實很重要，許多人常常忽略這個部分，當發生問題時已經來不及了。而ai牙刷的出現，我想對青年或老年都是一件好事情，按個按鈕，節省時間。

**Comment by larry40305** [comment]
現在AI常常被用在解決一些日常生活的小事，但這些產品的價格真的一點也不親民，應該要過很久才會慢慢普及。

**Comment by 0212Jerry** [comment]
看完突然想到，如果Ai 在刷牙的同時也能判斷牙齒的健康應該會是個很好的發明，在刷牙的同時，你的牙刷還可以幫你看你是否有蛀牙，是否有牙周病的前兆，發現了還可以及時處理，畢竟牙痛不是病，但痛起來要人命。不過在我看來，光是要讓探頭在口腔內的情況下正常運作就已經是一大難題了。

**Comment by Valentinaa0214** [comment]
價格實在是難以入手，這類的AI牙刷通常比傳統牙刷貴超多，並且需要一個應用程式來紀錄及提供反饋，產品的效果和耐用性也不確定，可能需要觀望一下，大部分的一般消費者可能也不太會買單。

**Comment by Vincent20030314** [comment]
AI結合牙刷更能幫助我們解決自己平常注意不到的地方，但價格有點難以負荷，希望以後產品出現時我有錢能購買。

**Comment by yjpxi** [comment]
飛利浦的智能牙刷能夠提供詳細的刷牙分析，甚至能通過手機應用來監控你的刷牙習慣，提醒你刷牙的時間和力度。
